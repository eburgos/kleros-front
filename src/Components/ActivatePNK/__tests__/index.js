import React from 'react'
import ActivatePNK from '../index'

describe('Activate Exist', () => {
  const component = shallow(<ActivatePNK />)
  test('should render Activate component correctly', () => {
    expect(component).toMatchSnapshot()
  })
})
