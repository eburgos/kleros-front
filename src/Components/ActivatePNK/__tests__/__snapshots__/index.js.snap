// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activate Exist should render Activate component correctly 1`] = `
<div
  className="ActivatePNK-container"
>
  <h1>
    Activate PNK
  </h1>
  <withRouter(Connect(ReduxForm)) />
</div>
`;
